docs/how-to/modify-kernel-boot-parameters.rst

```diff
@@ -0,0 +1,140 @@
+How to modify kernel boot parameters
+=======================================
+
+The real-time performance-tuning features compiled into the kernel can be
+controlled by various means. The approach used here is a common one:
+passing command line parameters via a bootloader.
+
+The Linux kernel supports a wide range of parameters, but only a few of them are relevant to real-time tuning.
+:doc:`/reference/kernel-boot-parameters` is good a reference.
+
+Depending on the underlying boot mechanism, there are multiple ways to pass command line parameters to the kernel.
+
+For Raspberry Pi, jump to :ref:`raspberry-pi`.
+
+Ubuntu Core uses GRUB but provides a different interface for 
+`modifying kernel options <https://ubuntu.com/core/docs/modify-kernel-options>`_.
+
+.. Replace the Ubuntu Core link to the internal how-to (https://documentation.ubuntu.com/real-time/en/ubuntu-core-devel/how-to/uc-boot-parameters/), once released.
+
+Continue reading for systems that use the GNU GRUB bootloader.
+
+Modify kernel boot parameters via GRUB
+--------------------------------------
+
+You can temporarily or permanently configure parameters which GRUB will pass to
+Real-time Ubuntu.
+
+When first testing a real-time tuning feature, `configure it temporarily
+<#configure-grub-temporarily>`_. Temporary bootloader parameters are passed to
+the kernel on a single startup. To re-use them you'll need to re-enter them ---
+they don't persist across reboots.
+
+When you're satisfied that a parameter achieves what you want, you can make it
+permanent by `updating GRUB's configuration <#configure-grub-permanently>`_
+then rebooting.
+
+Configure GRUB temporarily
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reboot, and interrupt GRUB's startup sequence at its bootloader menu.
+
+Highlight the entry --- usually **Ubuntu** --- that would boot Real-time
```

Comment:
Hey @farshidtz , is there a reason why you use 3 `---` instead of just a single dash `-`?
Thanks.

Reply:
I'm just refactoring this text, moving it from the CPU config doc to the modifying kernel config. The `---` creates an Em dash:
![image](https://github.com/canonical/real-time-ubuntu-docs/assets/11150423/d2ea7906-2ff3-4ec8-8d31-193d4fa92615)

Reply:
Oh, as long as the em dash was intentional.
thanks!